agent/consul/rpc.go

```diff
@@ -205,13 +205,6 @@ func canRetry(args interface{}, err error) bool {
 		return true
 	}
 
-	// If we were chunking and the term changed, resubmit against the new
-	// leader
-	intErr, ok := args.(error)
-	if ok && strings.Contains(intErr.Error(), raftchunking.ErrTermMismatch.Error()) {
-		return true
-	}
-
```

Comment:
Does this not make sense any more?

Reply:
Yeah I had a talk with Brian about how term changes would happen. Basically if a term change happened then somewhere along the way the client's applies would fail because it would no longer be the leader. When this happens the client will get an error and this type of error should always cause a retry (earlier in that function). The retry will use a new op num, so it seems like in practice we'd never actually see this error anyways.